krikri.gemspec

```diff
@@ -57,12 +57,12 @@ Gem::Specification.new do |s|
   # properly.
   s.add_dependency "bootstrap-sass", "3.3.4.1"
 
-  s.add_development_dependency 'krikri-spec', '~> 0.0'
-  s.add_development_dependency "sqlite3"
-  s.add_development_dependency "jettywrapper", '~> 2.0'
-  s.add_development_dependency "rspec-rails", '~> 3.3'
-  s.add_development_dependency 'webmock'
-  s.add_development_dependency 'factory_girl_rails', '~>4.4.0'
-  s.add_development_dependency 'pry-rails'
-  s.add_development_dependency 'timecop'
+  s.add_development_dependency "krikri-spec", "~> 0.0"
```

Comment:
I _think_ it should be `~> 0.0.1` so that it goes `0.0.2` ... `0.0.3` but not `0.1.0`.

I believe that the convention is that, for major version 0, which is assumed to be immature, you upgrade the tiny version always and treat it as if it were the minor version; and upping the minor version is saying "there may be breaking changes".